tox.ini

```
@@ -61,6 +61,6 @@ commands_pre =
     pip list
     pip check
 commands =
-    pytest --cov
+    pytest  xsdba  {posargs}
```

This is failing because there is no `xsdba` folder in the top-level (it lives in `src`). Also, `pytest` is looking for tests.

The `--cov` argument takes a keyword (`xsdba`) to know that the relevant module ton look at is the library. `--cov` on its own is an error.

```suggestion
    pytest {posargs}
```